app/screens/AppNavigator/screens/Dex/PoolSwap/PoolSwapScreen.tsx

```diff
@@ -85,7 +85,8 @@ export function PoolSwapScreen ({ route }: Props): JSX.Element {
         tokenA,
         tokenB,
         swap,
-        fee
+        fee,
+        poolpair
```

Comment:
where is this used?

Reply:
when cancelling, need to pass params again.

Reply:
https://github.com/DeFiCh/wallet/blob/141c024490291b2fda3979e39c5bfe4dcd260869/app/screens/AppNavigator/screens/Dex/PoolSwap/ConfirmPoolSwapScreen.tsx#L65